src/js/app.js

```
@@ -151,1263 +418,2195 @@ function validateIdeasUrl(ideasUrl) {
+  if (!name) return;
+  const nextBookmarks = new Set(bookmarkedSet);
+  if (shouldAdd) nextBookmarks.add(name);
+  else nextBookmarks.delete(name);
+  try {
+    localStorage.setItem('bookmarks', JSON.stringify([...nextBookmarks]));
+  } catch (e) {
+    console.warn('Bookmark persistence failed because browser storage is unavailable.', e);
+    return;
+  }
+  bookmarkedSet.clear();
+  nextBookmarks.forEach(n => bookmarkedSet.add(n));
```

Biome (2.5.0)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)